requirements/app.in

```diff
@@ -65,10 +59,8 @@ marshmallow-sqlalchemy>=0.23.1
 webargs
 # Minimum version that correctly aligns time series that include NaN values
 uniplot>=0.7.0
-# Maximum constraints here due to Flask-Classful not supporting Werkzeug 2.2.0 yet, see GH#595 and https://github.com/teracyhq/flask-classful/pull/145
-Flask-SQLAlchemy>=2.4.3,<3
+Flask-SQLAlchemy>=2.4.3
```

Comment:
According to https://github.com/marshmallow-code/flask-marshmallow/issues/262, I believe this minimum should raise to 2.5.2 ?